EMS-UI/src/main.ts

```diff
@@ -4,7 +4,7 @@ import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { AppModule } from './app/app.module';
 import { environment } from './environments/environment';
 
-if (environment.production) {
+if (environment.ENV_NAME !== 'local') {
```

Comment:
is the purpose of ENV_NAME variable just to check if we should be in prod mode or not? If so couldn't it just be a 'PROD_MODE' variable?

Reply:
Yep, that is its only purpose at the moment - I'm assuming you're suggesting it can be a `boolean` variable ('true'/'false'). It was a boolean variable before, but I assumed that would cause problem due to the substitution method.

```typescript
export const environment = {
ENV_NAME: '$ENV_NAME',
```

If this was a boolean this would become:
```typescript
export const environment = {
PROD_MODE: $IS_IN_PROD,
```
I'm assuming this would break, as `$IS_IN_PROD` is not a valid literal value or existing variable.